Templates/SendGrid-Timer-CSharp/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "defaultFunctionName": "SendGridTimerCSharp",
+    "description": "$SendGridTimerCSharp_description",
+    "name": "SendGrid",
+    "language": "C#",
+    "category": [
+        "$temp_category_samples",
+        "$temp_category_dataProcessing"
```

Comment:
We'd only want to add a template like this in samples, not in core dataProcessing